MP3DurationError is effectively private

In error.rs, MP3DurationError is declared public, but the module that contains it is private, so external crates are unable to reference the MP3DurationError type:
```
error[E0603]: module `error` is private
   --> src/errors.rs:163:32
    |
163 |     fn from(err: mp3_duration::error::MP3DurationError) -> Self {
    |
```

Either the error module should be made public, or the MP3DurationError type should be re-exported from lib.rs.

--- a/src/lib.rs
+++ b/src/lib.rs
@@ -16,6 +16,8 @@ use crate::constants::*;
 use crate::context::Context;
 use crate::error::*;
 
+pub use crate::error::MP3DurationError;
+
 fn get_bitrate<T: Read>(
     context: &Context<T>,
     version: Version,

